app/uk/gov/hmrc/helptosave/controllers/HelpToSaveController.scala

```diff
@@ -105,6 +106,26 @@ class HelpToSaveController @Inject() (val enrolmentStore:         EnrolmentStore
     implicit request ⇒ checkEligibility(nino, routes.HelpToSaveController.checkEligibility(nino).url)
   }
 
+  def doBarsCheck(): Action[AnyContent] = ggOrPrivilegedAuthorised { implicit request ⇒
+    request.body.asJson.map(_.validate[BarsRequest]) match {
+
+      case Some(JsSuccess(barsRequest, _)) ⇒
+        barsService.validate(barsRequest, request.uri).flatMap {
```

Comment:
note that the `request.uri` is always the BE uri `/help-to-save/validate-bank-details` irrespective of whether request originates from FE or API layer

Reply:
i mean its the path param for auditing